src/main/java/lotto/model/LottoRules.java

```diff
@@ -0,0 +1,41 @@
+package lotto.model;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LottoRules {
+
+    private final int START_NUM = 1;
```

Comment:
상수는 static을 붙여서 선언해주면 좋을 것 같아요!

```suggestion
    private static final int START_NUM = 1;
```